tools/openblas_support.py

```diff
@@ -67,13 +66,20 @@ def get_manylinux(arch):
 
 def download_openblas(target, plat, ilp64):
     osname, arch = plat.split("-")
+    # Auditwheel is the only environment variable that indicates musl. May want
+    # to add build identifier to env.
+    # Auditwheel: see: https://cibuildwheel.readthedocs.io/en/stable/options/
+    auditwheel_plat = os.environ.get("AUDITWHEEL_PLAT", "")
     fnsuffix = {None: "", "64_": "64_"}[ilp64]
     filename = ''
     headers = {'User-Agent':
                ('Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 ; '
                 '(KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.3')}
     suffix = None
-    if osname == "linux":
+    if "musllinux" in auditwheel_plat:
+        suffix = f"musllinux_1_1_{arch}.tar.gz"
+        typ = "tar.gz"
+    elif osname == "linux":
```

Comment:
Determining the glibc is a bit obscure. Perhaps this should undergo a deeper refactoring to better differentiate between musllinux and manylinux